src/UI/Actions.hs

```diff
@@ -285,18 +285,24 @@ setUnread =
     , _aAction = (liftIO . updateReadState Notmuch.addTags)
     }
 
-mailIndexUp :: Action 'BrowseMail AppState
-mailIndexUp =
+listUp :: Action m AppState
```

Comment:
Should we implement a class for *"mode with a navigable list"* and define it in terms of
that class?

Reply:
Yes I'd like this idea. Created #112 for this.